zarnitsa/DataAugmenterInternally.py

```diff
@@ -12,52 +12,38 @@ class DataAugmenterInternally(AbstractDataAugmenter):
     def __init__(self, n_jobs=1):
         self.n_jobs = n_jobs
 
-    def augment_dataframe(
-        self,
-        df: pd.DataFrame,
-        aug_type="normal",
-        freq=0.2,
-        return_only_aug=False,
-    ) -> pd.DataFrame:
+    def augment_dataframe(self, df: pd.DataFrame, **kwargs) -> pd.DataFrame:
```

Comment:
if you remove explicit args and use kwargs, then write about them in docstring, otherwise it is not obvious which args are allowed

Reply:
agree, i will fix it